src/vmm/src/arch/aarch64/mod.rs

```diff
@@ -123,6 +130,31 @@ fn get_fdt_addr(mem: &GuestMemoryMmap) -> u64 {
     layout::DRAM_MEM_START
 }
 
+/// Load linux kernel into guest memory.
+pub fn load_kernel(
+    kernel: &File,
+    guest_memory: &GuestMemoryMmap,
+) -> Result<EntryPoint, ConfigurationError> {
+    // Need to clone the File because reading from it
+    // mutates it.
+    let mut kernel_file = kernel
+        .try_clone()
+        .map_err(|_| ConfigurationError::KernelFile)?;
+
+    let entry_addr = Loader::load::<File, GuestMemoryMmap>(
```

Comment:
same here actually. no need for turbofish

Reply:
fixed